### PR TITLE
Peers subpage

### LINKED ISSUE #21: Admin subpage: Peers
Definition of done:
- no styling, plain html
- title: `Peers seen in the network (${peers.length})`
- table with columns: id (from 0 up), peerId, alias, Eligible, Multiaddrs
- refresh button needed at the same line as title of the page, button should be on the right

### PR DESCRIPTION
**Pull request by ItsLuized**
Closes #21

### Overview
This pull Request provides peers subpage to show the announces peers with their info

Things to be added:

- [x] no styling, plain html
- [x] title: Peers seen in the network (${peers.length})
- [x] table with columns: id (from 0 up), peerId, alias, Eligible, Multiaddrs
- [x] refresh button needed at the same line as title of the page, button should be on the right

To be noted:

Elegible is currently set to show true if the peerId is connected to the current node (in both the announced and connected arrays of peers)

### Current state
image

### CODE REVIEW

File: src/sections/peers.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        apiEndpoint: loginData.apiEndpoint!,
+        apiToken: loginData.apiToken!,
+      })
+    );
+    dispatch(
+      actionsAsync.getAliasesThunk({
+        apiEndpoint: loginData.apiEndpoint!,
+        apiToken: loginData.apiToken!,
+      })
+    );
```
Comment (camilovegag) [inline src/sections/peers.tsx]:
```ts
if (loginData) {
   handleRefresh();
}
```

File: src/sections/peers.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  };
+
+  return (
+    <Section>
+      <h2>
+        Peers seen in the network ({peers?.announced?.length || 0}){' '}
+        <button
+          onClick={() => {
+            handleRefresh();
+          }}
```
Comment (camilovegag) [inline src/sections/peers.tsx:56]: This can be changed just to :
```ts
onClick={handleRefresh}
```

File: src/sections/peers.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+              <TableCell>id</TableCell>
+              <TableCell>peerId</TableCell>
+              <TableCell>alias</TableCell>
+              <TableCell>Elegible</TableCell>
+              <TableCell>Multiaddrs</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {Object.entries(peers?.announced ?? {}).map(([id, peer], key) => (
+              <TableRow key={key}>
```
Comment (camilovegag) [inline src/sections/peers.tsx:75]: Using `id` as the `key` here might be better than using the second map argument `key` if we change the order of the array in the future

**Review by camilovegag — CHANGES_REQUESTED** [review]

File: src/sections/peers.tsx (hunk 1 above)
Comment (ItsLuized) [reply src/sections/peers.tsx]: Regarding this, in a different PR, this was implemented in a similar way and in the Review it was adviced to remove it:
https://github.com/hoprnet/hopr-admin/pull/40#discussion_r1221068542

**Review by camilovegag — APPROVED** [review]

**Pull request merged**